containers/ecr-viewer/src/app/components/EcrFilters.tsx

```diff
@@ -143,14 +156,26 @@ const FilterReportableConditions = ({
         {allConditions.length > 0 && (
           <div className="border-top-1px border-base-lighter margin-x-105"></div>
         )}
+
+        {/* No conditions reported */}
+        <CheckboxOptions
+          groupName="condition"
+          filterItems={noConditions}
+          onChange={handleCheckboxChange}
+        />
+
+        {Object.keys(noConditions).length > 0 && (
```

Comment:
could we put this conditional render around the whole chunk here to make clear that the checkbox also only appears when there are conditions?